src/components/Calculator.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/button-has-type */
+import React, { Component } from 'react';
+import Digits from './Digits';
+
+// implement calculator component
+class Calculator extends Component {
+  render() {
+    return (
+      <div className="calculator">
+        {/* display */}
+        <div className="display">
+          <span>0</span>
+        </div>
+        {/* main keys */}
+        <div className="main-keys">
+          <button>AC</button>
+          <button>+/-</button>
+          <button>%</button>
+        </div>
+
+        {/* operator keys */}
+        <div className="operator-keys">
+          <button>+</button>
+          <button>x</button>
+          <button>-</button>
+          <button>/</button>
+          <button>=</button>
+        </div>
+
+        {/* digit keys */}
+        <div className="digit-keys">
+          <Digits />
+          <div className="special-digits">
+            <button>0</button>
+            <button>.</button>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
```

Comment:
- [ ] Kindly fix your UI. As you can see from the provided design, the numeric buttons and the operations buttons are on the same line. Kindly fix that.

- [ ] Additionally, you swapped the positions of the divide and plus signs, contrary to what we have on the template. Kindly fix.

- [ ] Your divide sign should be "&#247;", not "/". Fix that as well. You can use this HTML entity code: `&#247;` if you don't have that divide sign on your keyboard.

  ![ui-calc](https://user-images.githubusercontent.com/36542986/185130616-c98e4338-56bf-49fe-96b3-4b8756bea1f7.jpg)